src/tree/values/nodes/Paragraph.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Represents a paragraph with text within an HTML-document that can be read by a reader.
+ */
+class Paragraph {
+	/**
+	 * A paragraph within an HTML-document that can be read by a reader.
+	 *
+	 * @param {TextContainer} textContainer That includes the text without any formatting and phrasing elements if any.
+	 * @param {string} openingTag The opening tag of the paragraph.
+	 * @param {string} closingTag The closing tag of the paragraph.
+	 */
+	constructor( textContainer, openingTag = "<p>", closingTag = "</p>" ) {
```

Comment:
Is `<p>` a wanted default? Or is an empty string more appropriate?

Reply:
I think `<p>` makes a lot of sense as a default, I do not think an empty string is more appropriate. We can keep `<p>` for now and change it later if we notice that it is not good.

Reply:
It is now `tag` with the default of an empty string. What made you change your mind?